tests/middleware/test_base.py

```diff
@@ -82,20 +82,21 @@ def test_custom_middleware(test_client_factory):
 
 
 def test_middleware_decorator(test_client_factory):
```

Comment:
Do we need to keep this test?
As the name suggests I think we're not really testing what it says. I tried to removed that I #1499 but I'll close it in favour of this.

Reply:
I don't think so, no.